src/errors/is-internal-stack-frame.ts

```diff
@@ -33,7 +34,7 @@ const INTERNAL_INCLUDES_PATH_SEGMENTS = [
 
 function isInternalFile (filename = ''): boolean {
     return !filename ||
-        !filename.includes(sep) ||
+        !(filename.includes(sep) || filename.startsWith(FILE_PROTOCOL_PREFIX)) ||
```

Comment:
Reuse [this](https://github.com/DevExpress/testcafe/blob/master/src/client/browser/index.js#L61) code.